src/data/selector.ts

```diff
@@ -111,8 +111,8 @@ export interface DateTimeSelector {
 export interface DeviceSelector {
   device: {
     integration?: string;
-    manufacturer?: string;
-    model?: string;
+    manufacturer?: string | [string];
+    model?: string | [string];
```

Comment:
```suggestion
    manufacturer?: string | string[];
    model?: string | string[];
```
Tech review : should be array of string